can_bus/quadruna/BRUSA/BRUSA_tx.json

```diff
@@ -0,0 +1,21 @@
+{
+    "Vitals": {
+        "msg_id": 1552,
+        "cycle_time": 100,
+        "description": "Flags relaying BMS Charger Faults",
+        "signals": {
+            "IsConnected": {
+                "start_bit": 7,
+                "bits": 1
+            },
+            "Error": {
+                "start_bit": 6,
+                "bits": 1
+            },
+            "PowerLimitedWarning": {
+                "start_bit": 5,
+                "bits": 1
+            }
+        }
+    }
```

Comment:
Not sure if this worked for you guys but this failed to decode the signals from the Brusa when we tried it yesterday. Also, there's a few other messages that aren't on here, maybe add a TODO to support them later?

Reply:
Sounds good, it just involves decoding the entire can msg so that the signals can be properly displayed on pcan. There are quite a few signals in the msg though so will be done at a later time. Added note in chargestate.h